### PR TITLE
Fix:10071 - Accept watchOptions.aggregateTimeout be 0

### PR DESCRIPTION
**Pull request by jeffin143**
Handles : [#10071](https://github.com/webpack/webpack/issues/10071)

Aligned with 
https://github.com/webpack/webpack/pull/10118

### CODE REVIEW

File: lib/watchpack.js [hunk 1]
```diff
@@ -64,7 +64,9 @@ class Watchpack extends EventEmitter {
 	constructor(options) {
 		super();
 		if (!options) options = {};
-		if (!options.aggregateTimeout) options.aggregateTimeout = 200;
+		if (typeof this.watchOptions.aggregateTimeout !== "number") {
+			this.watchOptions.aggregateTimeout = 200;
```
Comment (simonhowey) [inline lib/watchpack.js]: I think you'll want to change watchOptions -> options here.

Comment (jeffin143) [reply lib/watchpack.js]: Yes indeed, Thanks for pointing out :rocket:

**Comment by jeffin143** [comment]
friendly-ping @sokra , bump

**Comment by sokra** [comment]
Thanks

**Pull request merged**